lib/Doctrine/ruleset.xml

```diff
@@ -181,6 +181,8 @@
     <rule ref="SlevomatCodingStandard.Classes.SuperfluousInterfaceNaming"/>
     <!-- Forbid suffix "Trait" for traits -->
     <rule ref="SlevomatCodingStandard.Classes.SuperfluousTraitNaming"/>
+    <!-- Forbid "@inheritdoc" in favor of "@inheritDoc" annotation -->
```

Comment:
Btw the sniff checks a lot more annotations :)

Reply:
Yes, of course. But it was added here just for `@inheritDoc`.

Reply:
Yes, but the comment is not true because the sniff is not configured to solve just “inheritDoc”

Reply:
That's right. I guess I can improve it.